dist/static/site.css

```diff
@@ -0,0 +1,366 @@
+}
+
+body {
+  font-family: "Source Sans Pro", sans-serif;
+  color: #fff;
+}
+
+header {
+  text-align: center;
+}
+
+#heading {
+  font-family: "Fascinate Inline", monospace;
+  font-size: 3em;
+  user-select: none;
+  margin: 0;
+  padding: 30px;
+}
+
+h1,
+h2 {
+  text-shadow: 2px 2px 4px #000000;
+}
+
+h2 {
+  font-weight: 600;
+}
+
+@keyframes pop-in {
+  from {
+    transform: translateY(-20px);
+    opacity: 0;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 1;
+  }
+}
+
+#nameentry {
+  position: absolute;
+  top: 20vh;
+  left: 20vw;
+  background: #111;
+  padding: 15px 30px;
+  border-radius: 5px;
+  box-sizing: border-box;
+  width: 60vw;
+  animation: 250ms pop-in ease;
+}
+
+button,
+input {
+  font-family: "Source Sans Pro", sans-serif;
+}
+
+input[type="text"] {
+  font-family: monospace;
+  background: #1e1e1e;
+  color: #fff;
+  border-radius: 5px;
+  border: 2px solid transparent;
+  padding: 5px 8px;
+  outline: none;
+  box-sizing: border-box;
+  transition: 40ms ease-in-out border-color;
+}
+
+input:focus {
+  background: #222;
+  border: 2px solid #a50bca;
+}
+
+#nameentry input {
+  font-size: 28px;
+  width: 100%;
+  padding: 10px 15px;
+  margin: 10px 0;
+}
+
+button {
+  background: transparent;
+  color: #888;
+  font-size: 16px;
+  font-weight: 600;
+  margin: 8px 0;
+  padding: 10px;
+  border: none;
+  cursor: pointer;
+  transition: 120ms ease-in-out;
+  transition-property: background-color padding color;
+}
+
+button:after {
+  content: "\2192";
+  transition: 120ms ease-in-out;
+  margin-left: 6px;
+}
+
+button:hover:after {
+  margin-left: 10px;
+}
+
+button:hover {
+  background: #a50bca;
+  color: #fff;
+  padding: 10px 20px;
+}
+
+main {
+  margin: 0 16vw;
+}
+
+.song {
+  display: flex;
+  height: 90px;
+  background-color: rgba(0, 0, 0, 0.5);
+  border-radius: 5px;
+  overflow: hidden;
+  align-items: center;
+}
+
+.name {
+  text-shadow: 2px 2px 4px #000000;
+  font-family: "Righteous", monospace;
+}
+
+#nowplaying {
+  cursor: default;
+}
+
+#nowplaying .song {
+  width: 100%;
+  overflow: hidden;
+  border: none;
+  background: #111;
+  height: 90px;
+  align-items: center;
+}
+
+#nowplaying .song > .songtitle {
+  font-size: 1.2em;
+  font-family: "Righteous", monospace;
+  max-width: none;
+}
+
+#nowplaying button {
+  align-self: center;
+}
+
+@keyframes push-desc-out {
+  from {
+    transform: translateX(0px);
+  }
+  100% {
+    transform: translateX(30px);
+  }
+}
+
+#currentsong {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+#currentplayer-descr {
+  font-family: "Righteous", monospace;
+  animation: 250ms pop-in ease, 400ms push-desc-out ease 250ms;
+  padding: 10px 0;
+  position: relative;
+  transform: translateX(30px);
+  text-shadow: 2px 2px 4px #000000;
+}
+
+@keyframes rotating-disc {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+@keyframes fade-in {
+  from,
+  40% {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
```

Comment:
Not sure what is here. Do you want it to fade in starting at 40% or from the very beginning?
```css
@keyframes fade-in {
    from {
        opacity: 0;
    }
    to {
        opacity: 1;
    }
}
```css

Or

@keyframes fade-in {
    40% {
        opacity: 0;
    }
    100% {
        opacity: 1;
    }
}
```

Reply:
This is a better question for @encadyma

Reply:
It's a delayed fade in. I think the rotating disc is the only one that uses the animation at the moment, so it's fine to start it from `0%`/`from` and use the `animation` statement to add a delay to `fade-in`.